lib/json-schema/schema/reader.rb

```diff
@@ -58,6 +84,7 @@ def initialize(options = {})
       # @raise [JSON::Schema::ReadRefused] if +accept_uri+ or +accept_file+
       #   indicated the schema could not be read
       # @raise [JSON::Schema::ParseError] if the schema was not a valid JSON object
+      # @raise [JSON::Schema::ReadFailed] if read fails
```

Comment:
I'd extend the comment to also cover that this only occurs if reading the schema is permitted but the actual attempt to retrieve it fails.
As it is now, it appears to me like an explanation for wet water, telling it is wet :wink: